src/spdx/writer/tagvalue/file_writer.py

```diff
@@ -28,8 +27,9 @@ def write_file(file: File, text_output: TextIO):
     for file_checksum in file.checksums:
         write_value("FileChecksum", write_checksum_to_tag_value(file_checksum), text_output)
 
-    write_license_expression("LicenseConcluded", file.license_concluded, text_output)
-    write_license_expression("LicenseInfoInFile", file.license_info_in_file, text_output)
+    write_value("LicenseConcluded", file.license_concluded, text_output)
+    for license_info in file.license_info_in_file:
+        write_value("LicenseInfoInFile", license_info, text_output)
```

Comment:
This causes a TypeError if snippet.license_info_in_snppet is SpdxNone or SpdxNoAssertion.

Reply:
fixed